### PR TITLE
Add initial VIA support for gherkin and pockettype

### PR DESCRIPTION
**Pull request by sirdicholas15**
40percentclub gherkin and maartenwut pockettype

Added the requred files needed for initial VIA support for the Gherkin and PocketType

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
pockettype has been moved to <https://github.com/qmk/qmk_firmware/tree/master/keyboards/evyd13/pockettype>

Also licence headers need to be added to the keymap.c files

**Comment by Suschman** [comment]
Can the USB ids for gherkin be aligned with the other enabled 40percentclub boards before it goes into VIA?

#define VENDOR_ID       0x3430 // "40"
#define PRODUCT_ID       0x4748 // "GH"

> It is recommended to choose a value of VENDOR_ID that unique to the keyboard PCB’s designer/vendor, i.e. it will be the same for all keyboards with a common parent directory.

File: keyboards/40percentclub/gherkin/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/40percentclub/gherkin/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/evyd13/pockettype/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/evyd13/pockettype/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**